File: VB/App_Code/MySavedObjects.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.Web

Public Class MySavedObjects
    Public Sub New()
        FileName = String.Empty
        Url = String.Empty
    End Sub

    Public Property RowNumber() As Integer

    Public Property Url() As String

    Public Property FileName() As String
End Class

File: CS/App_Code/MySavedObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.Web;

public class MySavedObjects
{
    public MySavedObjects()
    {
        FileName = String.Empty;
        Url = String.Empty;
    }

    public int RowNumber {
        get; set;
    }

    public string Url {
        get; set;
    }

    public string FileName {
        get; set;
    }
}